app/graph/mutations/graphql_crud_operations.rb

```diff
@@ -7,7 +7,20 @@ def self.safe_save(obj, attrs, parent_names = [])
       obj.send(method, value) if obj.respond_to?(method)
     end
     obj.disable_es_callbacks = Rails.env.to_s == "test"
-    obj.save_with_version!
+
+    begin
+      obj.save_with_version!
+    rescue RuntimeError => e
+      if e.message.include?("This item already exists") &&
```

Comment:
Request: @vasconsaurus , this is not a good way to check for the error because of two reasons:

(1) If `I18n.locale` is not English, then this error message will be localized, so won't match this condition (for example, for Portuguese it will be "Este item já existe").

(2) The copy for the error message may change in the future.

I suggest you check for the exception class or some other non-localized information.

Reply:
@caiosba, questions:
- The error we get is a `RuntimeError`, so it's not super specific. It does send the lapis code in the message, so I changed it to look for that. ([here](https://github.com/meedan/check-api/pull/2079/commits/cb97946cc90cf337402d2424899a20387f085a9a))
- However, would it be worth for us to create a custom error? We get the error from here:
https://github.com/meedan/check-api/blob/cb97946cc90cf337402d2424899a20387f085a9a/app/models/concerns/project_association.rb#L139-L155

Reply:
@vasconsaurus , I don't think we need a specific code, I think that `DUPLICATED` is good enough for this case. But please use the variable `LapisConstants::ErrorCodes::const_get('DUPLICATED')` instead of the hard-coded value "9".